electron_app/src/aliceManager.js

```diff
@@ -0,0 +1,81 @@
+const { spawn } = require('child_process');
+const path = require('path');
+const dbManager = require('./DBManager');
+const portscanner = require('portscanner');
+
+const getAlicePath = nodeEnv => {
+  switch (nodeEnv) {
+    case 'development': {
+      return path.join(__dirname, '../../signal_interface/build/Release/alice');
+    }
+    default: {
+      return path.join(path.dirname(__dirname), 'extraResources', 'alice');
```

Comment:
Is this path relative to `electron-starter.js` or `src` folder?
Check it out :)